src/app/Scenes/MyCollection/MyCollection.tsx

```diff
@@ -240,10 +255,12 @@ export const MyCollectionContainer = createPaginationContainer(
         id
         myCollectionInfo {
           includesPurchasedArtworks
-          artistsCount
           artworksCount
         }
         ...MyCollectionCollectedArtists_me
+        userInterestsConnection(first: 10, category: COLLECTED_BEFORE, interestType: ARTIST) {
```

Comment:
Maybe this works even without fetching any user interests. Like this Gravity would send less data to Metaphysics.

```suggestion
        userInterestsConnection(category: COLLECTED_BEFORE, interestType: ARTIST) {
```

Reply:
Yeh, I tried. It highlights all other usages of this connection with an error
> Expected all fields on the same parent with the name or alias `userInterestsConnection` to have the same argument values after applying fragment arguments. This field has the applied argument values: (first: 10, category: COLLECTED_BEFORE, interestType: ARTIST)

Reply:
Ah, sure! They need to have the same arguments.